mkw_stats_bot/scripts/utilities/migrate_increase_war_count_precision.py

```
@@ -0,0 +1,132 @@
+                    MAX(war_count) as max_wars,
+                    MIN(war_count) as min_wars,
+                    AVG(war_count) as avg_wars,
+                    COUNT(*) as total_players
+                FROM players
+                WHERE is_active = TRUE;
+            """)
+
+            stats = cursor.fetchone()
+            if stats:
+                max_wars, min_wars, avg_wars, total_players = stats
+                print(f"\n📊 Current War Statistics:")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)